src/pages/home.tsx

```diff
@@ -0,0 +1,181 @@
+import styled from 'styled-components';
+import Head from 'next/head';
+import { FONT_STYLES } from '@src/styles/fontStyle';
+import NewsList from '@src/components/common/NewsList';
+import { COLOR } from '@src/styles/color';
+import axios from 'axios';
+import { useEffect, useState } from 'react';
+
+export interface videoType {
+  id: number;
+  title: string;
+  channel: string;
+  category: string;
+  date: string;
+  thumbnail: string;
+  isLiked: boolean;
+}
+
+function Home() {
+  const [videoList, setVideoList] = useState<videoType[]>([
+    {
+      id: 1,
+      title: '비트코인, 한때 1만 8천 달러 붕괴',
+      channel: 'SBS 뉴스',
+      category: '사회',
+      date: '2022.06.19',
+      thumbnail: 'https://이미지링크',
+      isLiked: true,
+    },
+    {
+      id: 2,
+      title: '비트코인, 한때 1만 8천 달러 붕괴',
+      channel: 'SBS 뉴스',
+      category: '사회',
+      date: '2022.06.19',
+      thumbnail: 'https://이미지링크',
+      isLiked: false,
+    },
+    {
+      id: 3,
+      title: '비트코인, 한때 1만 8천 달러 붕괴',
+      channel: 'SBS 뉴스',
+      category: '사회',
+      date: '2022.06.19',
+      thumbnail: 'https://이미지링크',
+      isLiked: false,
+    },
+    {
+      id: 4,
+      title: '비트코인, 한때 1만 8천 달러 붕괴',
+      channel: 'SBS 뉴스',
+      category: '사회',
+      date: '2022.06.19',
+      thumbnail: 'https://이미지링크',
+      isLiked: false,
+    },
+    {
+      id: 5,
+      title: '비트코인, 한때 1만 8천 달러 붕괴',
+      channel: 'SBS 뉴스',
+      category: '사회',
+      date: '2022.06.19',
+      thumbnail: 'https://이미지링크',
+      isLiked: false,
+    },
+    {
+      id: 6,
+      title: '비트코인, 한때 1만 8천 달러 붕괴',
+      channel: 'SBS 뉴스',
+      category: '사회',
+      date: '2022.06.19',
+      thumbnail: 'https://이미지링크',
+      isLiked: false,
+    },
+    {
+      id: 7,
+      title: '비트코인, 한때 1만 8천 달러 붕괴',
+      channel: 'SBS 뉴스',
+      category: '사회',
+      date: '2022.06.19',
+      thumbnail: 'https://이미지링크',
+      isLiked: false,
+    },
+    {
+      id: 8,
+      title: '비트코인, 한때 1만 8천 달러 붕괴',
+      channel: 'SBS 뉴스',
+      category: '사회',
+      date: '2022.06.19',
+      thumbnail: 'https://이미지링크',
+      isLiked: false,
+    },
+  ]);
+
+  // const fetchVideoList = async () => {
+  //   try {
+  //     const { data } = await axios.get('https://5bf61531-1c07-442d-b743-28471f964f44.mock.pstmn.io/recommend_news');
+  //     setVideoList(() => data.data.videoList);
+  //   } catch (e) {
+  //     console.log(e);
+  //   }
+  // };
+
+  // useEffect(() => {
+  //   fetchVideoList();
+  // }, [videoList]);
+
+  if (!videoList) return null;
+  return (
+    <>
+      <Head>
+        <title>DeliverBle</title>
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+      <StHome>
+        <StBannerText>
+          <h1>
+            우리는 말하는 법은 배웠지만,
+            <br />잘 말하는 법은 배우지 못했다!
+          </h1>
+          <p>딜리버블과 함께 잘 말하는 법을 배워봐요!</p>
+        </StBannerText>
```

Comment:
이것도 
```suggestion
        <StBannerText>
          <h1>우리는 말하는 법은 배웠지만,</h1>
          <h1>잘 말하는 법은 배우지 못했다!</h1>
          <p>딜리버블과 함께 잘 말하는 법을 배워봐요!</p>
        </StBannerText>
```
이렇게 바꾸고, margin 속성이랑 & > h1 , & > p 이렇게 속성을 주면 스타일링 하기 쉬울 것 같아요!

Reply:
br 태그를 남용하는 게 안좋다는 것은 알고 있는데 `우리는 말하는 법은 배웠지만, 잘 말하는 법은 배우지 못했다!` 이게 한 문장이어서 h1를 두 개 사용해서 줄 바꿈을 하는 게 맞는 건지 고민이 되네요.
`우리는 말하는 법은 배웠지만,`과 `잘 말하는 법은 배우지 못했다!`의 스타일링이 다르게 들어간다면 제안주신 코드가 스타일링 관점에서는 더 편할 것 같지만 현재는 스타일링이 다르지 않기 때문에 단순 줄 바꿈이 의미상 더 적절한 것 같습니다.